Create Blog Post “devcity-comes-to-london” (#2257)
Nexmo/deved-platform: content/blog/en/devcity-comes-to-london.md

---

> title: DevCity comes to London!

Reviewer: We use a style capitalisation tool: https://capitalizemytitle.com/style/AP/#
```suggestion
title: Devcity Comes to London!
```

Reviewer 2: The name of the event is "DevCity".

description: After an explosive launch at Dreamforce in San Francisco, the
  DevRel team at Vonage are excited to share that their DevCity event series is
  coming to London!
author: lucinda-bone
published: true
published_at: 2022-09-27T12:01:14.677Z
updated_at: 2022-09-27T12:01:14.692Z
category: event
tags:

>   - "#event"

Reviewer: ```suggestion
  - event
```

comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---

> After an explosive launch at Dreamforce in San Francisco, the DevRel team at Vonage are excited to share that their DevCity event series is coming to London on Wednesday 26th October! Read on to find out how you can get involved in the latest Developer hangout on the block.

Reviewer 2: Let's not use words such as "explosive". Also, launch of what? Let's also expand "DevRel" to "Developer Relations" .
This sentence seems a bit convoluted.

> **What is DevCity?**

Reviewer: ```suggestion
## What Is DevCity?
```

> Hosted by the Vonage Developer Relations team, the DevCity event series brings together engaging content, international networking, exclusive swag and brilliant fun and games - all tailored for developers. Whether you’re new on the Dev scene or an experienced old timer, there is something for you. 

Reviewer: This sentence seems incorrect - brilliant fun and games? What is brilliant?

Reviewer 2: "experienced old timer" :D  Let's not use age related adjectives.

> Featuring a mix of Vonage speakers and special guests, the DevCity content will equip you with everything you need to know about integrating our Communications APIs into your projects - be it voice, video, SMS or verify. We’ll also be meeting happy Vonage customers, analysing the latest trends and demonstrating first hand how you can supercharge your communications. 

Reviewer: ```suggestion
Featuring a mix of Vonage speakers and special guests, the DevCity content will equip you with everything you need to know about integrating our Communications APIs into your projects - be it voice, video, SMS or verify. We’ll also be meeting happy Vonage customers, analysing the latest trends and demonstrating first-hand how you can supercharge your communications. 
```

> Come for the content, but stay for the party - we’ll be offering a unique range of activations, games and competitions, as well as the chance for you to get your hands on the latest swag that everyone’s talking about. 

Reviewer: What are activations?

Reviewer 2: "Come for the content, but stay for the party" => "Come for the tech, and stay for the party"

Reviewer 2: "activations" is probably activities. Could we make it more specific and expose the benefits for developers?

> Come for the content, but stay for the party - we’ll be offering a unique range of activations, games and competitions, as well as the chance for you to get your hands on the latest swag that everyone’s talking about. 

Reviewer: ```suggestion
Come for the content, but stay for the party - we’ll be offering a unique range of activities, games and competitions, as well as the chance for you to get your hands on the latest swag that everyone's talking about. 
```

> **Sounds great! So what’s the plan for London?**

Reviewer: ```suggestion
## Sounds Great! So What’s the Plan for London?
```

Reviewer 2: Could we change this to not sound in the 3rd person? I suggest: "Sounds great?! Here is the Plan for London

> DevCity London puts a new twist on an old favourite - bingo night! Join us at [Dabbers Social Bingo](<https://dabbers.bingo/>) from 6pm till 10pm GMT on Wednesday 26th October, where we’ll be serving up fresh content, delicious food and yummy cocktails. 

Reviewer: ```suggestion
DevCity London puts a new twist on an old favourite - bingo night! Join us at [Dabbers Social Bingo](https://dabbers.bingo) from 6 pm till 10 pm GMT on Wednesday 26th October, where we’ll be serving up fresh content, delicious food and yummy cocktails. 
```

Reviewer 2: change "from 6pm till 10pm GMT" to "from 6 till 10pm BST". Clocks change to GMT on Oct 30th.

> On the mic will be Chris Tankersley, Vonage’s Senior Dev Relations Engineer, giving you the low down on the Vonage API’s, followed by Zachary Powell, Senior Android Developer Advocate and Ashley Arthur, Node.js Developer, who will be talking about how Vonage moved its Client SDKs to Kotlin Multiple Platform and saved the world (kind of). Stay tuned for a special customer story announcement too. 

Reviewer: ```suggestion
On the mic will be Chris Tankersley, Vonage’s Senior Dev Relations Engineer, giving you the low down on the Vonage APIs, followed by Zachary Powell, Senior Android Developer Advocate and Ashley Arthur, Node.js Developer, who will be talking about how Vonage moved its Client SDKs to Kotlin Multiple Platform and saved the world (kind of). Stay tuned for a special customer story announcement too. 
```

Reviewer 2: "and saved the world (kind of). Stay tuned for a special customer story announcement too." to "and the benefits and challenges encountered along the way, as well as a special customer story".

> After the talks its bingo time! With great prizes to be won and unique challenges to conquer you’ll be grabbing your dabber and getting stuck in while the DJ blasts great tunes.

Reviewer: ```suggestion
After the talks, it's bingo time! With great prizes to be won and unique challenges to conquer, you’ll be grabbing your dabber and getting stuck in while the DJ blasts great tunes.
```

> Attending Droidcon London the next day? Vonage are bronze sponsors there, so stop by our stand to catch up with the team and get your hands on the Android Collectibles that were so popular in New York and Berlin.

Reviewer 2: Can we move this to a separate section, make it a bit "louder" and include a picture of the collectibles (we can use some pictures we took in Berlin and New York.

> **I love bingo! How do I get involved?**

Reviewer: ```suggestion
## I Love Bingo! How Do I Get Involved?
```

> [Register FREE online here](https://events.vonage.com/event/192e9dfd-a405-4dbd-8ad9-6f96e0eda9b6/summary) Spaces are limited, so we encourage you to register now to avoid disappointment! Dabbers Social Bingo is located at 18-22 Houndsditch, London EC3A 7DB, within a convenient five-minute stroll of both Aldgate and Liverpool Street Stations.

Reviewer: ```suggestion
[Register FREE online here](https://events.vonage.com/event/192e9dfd-a405-4dbd-8ad9-6f96e0eda9b6/summary). Spaces are limited, so we encourage you to register now to avoid disappointment! Dabbers Social Bingo is located at 18-22 Houndsditch, London EC3A 7DB, within a convenient five-minute stroll of both Aldgate and Liverpool Street Stations.
```

> Be sure to [follow us on Twitter](https://twitter.com/VonageDev) [](https://twitter.com/VonageDev)for all the latest event updates, and join us on our [Vonage Community Slack](https://developer.vonage.com/slack) to join the conversation and keep the networking flowing. Feel free to share the invite with your Dev friends too!

Reviewer: ```suggestion
Be sure to [follow us on Twitter](https://twitter.com/VonageDev) for all the latest event updates, and join us on our [Vonage Community Slack](https://developer.vonage.com/slack) to join the conversation and keep the networking flowing. Feel free to share the invite with your Dev friends too!
```

> **Where will you be next?**

Reviewer: ```suggestion
## Where Will You Be Next?
```

> After DevCity London we have our sights set on Paris, so keep an eye out for exciting news about that event coming very soon. We’ll be adding even more cities to the list for 2023 and we hope you will join us around the world!

Reviewer: ```suggestion
After DevCity London, we have our sights set on Paris, so keep an eye out for exciting news about that event coming very soon. We’ll be adding even more cities to the list for 2023, and we hope you will join us around the world!
```
